script.js

```diff
@@ -29,6 +56,8 @@ function clearCards(event) {
 
 function showCard(event) {
   count++;
+  click++;
+  turns.innerHTML = "TURNS: " + click / 2;
```

Comment:
I would probably change what is displayed here. I tihnk the logic is right but you probably don't need to show the user that they are 1/2 way through a turn